cloud_blobstore/__init__.py

```diff
@@ -142,6 +143,19 @@ def get(self, bucket: str, key: str) -> bytes:
         """
         raise NotImplementedError()
 
+    def get_all_metadata(
```

Comment:
We shouldn't expose this.  The semantics on each cloud are not quite identical.

Reply:
That's true - I didn't consider abuse by the Client. I'll remove this.